modules/300-prometheus/templates/grafana/secret-datasources-list.yaml

```diff
@@ -19,11 +19,6 @@ secureJsonData:
 apiVersion: 1
 
 deleteDatasources:
-- name: trickster
```

Comment:
This is a list of data sources pending deletion. Generally, this list includes data sources that have already been created and are scheduled for removal. You don't need to remove the Trickster data source from here.